### PR TITLE
question in rate limiting plugin

### PR DESCRIPTION
**Pull request by lmhmhl**
// Describe your PR here; eg. Fixes #issueNo
Question: When I test rate limit plugin, nowhere can pass the replenish rate and burst capacity, and there is no initial value.
First, In rate_limiter plugin, there is no initial value of burst_capatity and replenish_rate, so I add the initial Enum value in the constant file. Second, In soul admin, the rate_limit configuration pages can not configure burst_capatity and replenish_rate, so you should find some one familiar with JS adding the setting of burst_capatity and replenish_rate and log options in rate_limiter plugin rule configuration pages.

Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/RateLimiterPlugin.java [hunk 1]
```diff
@@ -22,10 +22,10 @@
 import org.dromara.soul.common.dto.convert.RateLimiterHandle;
 import org.dromara.soul.common.enums.PluginEnum;
 import org.dromara.soul.common.utils.GsonUtils;
-import org.dromara.soul.plugin.api.result.SoulResultEnum;
-import org.dromara.soul.plugin.base.utils.SoulResultWrap;
 import org.dromara.soul.plugin.api.SoulPluginChain;
+import org.dromara.soul.plugin.api.result.SoulResultEnum;
```
Comment (yu199195) [inline soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/RateLimiterPlugin.java:26]: why modify this?

File: soul-plugin/soul-plugin-ratelimiter/src/main/resources/META-INF/scripts/request_rate_limiter.lua [hunk 2]
```diff
@@ -64,4 +64,4 @@ end
 redis.call("setex", tokens_key, ttl, new_tokens)
 redis.call("setex", timestamp_key, ttl, now)
 
-return { allowed_num, new_tokens }
+return { allowed_num, new_tokens }
```
Comment (yu199195) [inline soul-plugin/soul-plugin-ratelimiter/src/main/resources/META-INF/scripts/request_rate_limiter.lua:67]: why modify this?

**Comment by yu199195** [comment]
why modify  js in static?

File: soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/RateLimiterPlugin.java (hunk 1 above)
Comment (lmhmhl) [reply soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/RateLimiterPlugin.java:26]: It is the same, but the order change, my bad

**Review by lmhmhl — COMMENTED** [review]
I am not familiar with it .

**Comment by lmhmhl** [comment]
Some thing wrong happened, I am so sorry, I delete my Static file, and re download the file, then cause the inconsistancy, I do not know how to fix it.

**Comment by yu199195** [comment]
@lmhmhl  hi， you can follow this ： [How contribute to Soul](https://dromara.org/website/en-us/docs/soul/contributor.html).

Look forward to your contribution again.

**Comment by yu199195** [comment]
@lmhmhl this pr have more error ,please close it..

**Pull request closed without merging**